www/docs/intro.md

```diff
@@ -307,8 +308,13 @@ it("should estimate fee", async function () {
 it("should forward to the implementation contract", async function () {
     const implementationFactory = await starknet.getContractFactory("contract");
     const account = ...;
-    const implementationClassHash = await account.declare(implementationFactory);
+    const txHash = await account.declare(implementationFactory);
 
+    // ensure that tx has been successful
+    const txReceipt = await starknet.getTransactionReceipt(txHash);
+    expect(txReceipt.status).to.equal("ACCEPTED_ON_L2" || "ACCEPTED_ON_L1");
```

Comment:
How should this work? Shouldn't some waiting have to be done? This is related to my other comment about potentially reverting to awaiting inside of `declare`.